infrastructure/lambda/template.yaml

```diff
@@ -108,9 +118,17 @@ Mappings:
       integration: 2
       production: 2
 
+  PrivatePkiMapping:
```

Comment:
Can this be removed given the work used to turn of ssl for perf testing?

Reply:
Removed and re-aligned with the variables used currently on main